news/2022-08-25-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,142 @@
+
+As well as this, specific nerfs for specfic patterns were implemented within a 'cheese detector', which was something that the osu!taiko pp committee wanted to stay away from. As such, a policy for the all the new reworks under our wing was formed.
+
+**Never nerf a specific playstyle or pattern solely for a lack of particular reason.**
+
+#### The new system of Stamina
+
+In this section, the keybinds relating to the two colours found within osu!taiko are named k<sup>1</sup>, k<sup>2</sup> and d<sup>1</sup>, d<sup>2</sup>.
+
+- Instead of assuming a full-alternate play style, we assume that **each note of the same colour is always alternated.** For example, for the pattern kddkddk, we assume the kat's will be hit by **k<sup>1</sup>** d<sup>1</sup> d<sup>2</sup> **k<sup>2</sup>** d<sup>1</sup> d<sup>2</sup> **k<sup>1</sup>**, and the don's will be hit by k<sup>1</sup> **d<sup>1</sup> d<sup>2</sup>** k<sup>2</sup> **d<sup>1</sup> d<sup>2</sup>** k<sup>2</sup>.
+- The reason for this is to assume the best case scenario for repeated keystrokes regardless of play style. In other words, it aims to approximate the theoretical minimum stamina required to play a given pattern with any playstyle, with any theoretically rollable pattern being rolled. We can’t (feasibly), nor is it desired to, detect playstyles, so we assume the case that requires the least physical repetition, which is to switch keys each time a colour is to be hit, and each play style will have their respective weaknesses.
+- The combined of these changes also helps with the cases where specific playstyles or patterns were nerfed in the previous rework, as we don't detect any patterns, just the pure stamina requirement for each map.
+
+### Readdressing Colour
+
+The old colour system detected the frequency of colour changes and had penalties for specific repeated patterns, this meant that only notes with a different colour from the previous could get more than a non-zero strain, which contributed to issues of complex patterns containing large groups of single coloured notes not being considered properly in the strain system.
+
+Similar to stamina, a repetition penalty was applied on durations of single-coloured patterns. While only the 5 most recent durations were considered in this penalty, if any two or more of those patterns were repeated, a harsh penalty was applied.
+
+Once again, going against the new policies of this rework, changes were made.
+
+#### The new Colour system
+
+For the new system we define colour difficulty as the amount of information contained within a given pattern (or sequence if you will). For example, we can take a look with the following colour patterns and the very rough amount of information required to describe them:
+
+![](/wiki/shared/news/2022-08-25-changes-to-osu-taiko-star-rating-and-performance-points/colour-patterns.jpg)
```

Comment:
Caption of figure should read _Superscript numbers represent the **number** of **notes**, and C represents a colour change._

Reply:
Applied in c03fa161c742a7a9051f00c3821e74cbb8cfdba5